README.md

```diff
@@ -16,17 +16,26 @@ Lightweight, zero-dependency, self-configuration for FreeBSD droplets on Digital
 >*WARNING: Any custom networking scripts you may have created in /etc/rc.conf.d will be replaced. Backup if needed.*
 
 Droplets built from DigitalOcean's base FreeBSD images have extra packages and scripts that will no longer be required for auto configuration. They should be disabled first to avoid conflicts. When you're confident your cleaned-up droplet is working properly with this solution, you can remove the preinstalled stuff you do not need:
+
+Unecessary packages:
+
 - avahi
 - bsd-cloudinit
 - python
 - curl
-- Related settings in /etc/rc.conf and files in /usr/local/etc
+
+`pkg delete avahi bsd-cloudinit python27 curl`
+
+Left over users:
+- `rmuser -y avahi avahi-autoipd messagebus`
```

Comment:
When you uninstall the packages there are messages about cleaning up these users.